org.jacoco.core/src/org/jacoco/core/internal/Java9Support.java

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2016 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.internal;
+
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.Opcodes;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Patching for Java 9 classes, so that ASM can read them.
+ */
+public final class Java9Support {
```

Comment:
Basically we require to have JavaDoc on all public Java classes and members.
* Javadoc: Missing comment for public declaration	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 25	Java Problem
* Javadoc: The method readClass(InputStream, boolean) from the type ClassReader is not visible	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 31	Java Problem
* Javadoc: Missing tag for return type	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 33	Java Problem
* Javadoc: Missing tag for parameter is	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 33	Java Problem
* Javadoc: Missing tag for parameter close	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 33	Java Problem
* Javadoc: Missing tag for declared exception IOException	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 34	Java Problem
* Javadoc: Missing comment for public declaration	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 79	Java Problem
* Javadoc: Missing comment for public declaration	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 83	Java Problem
* Javadoc: Missing comment for public declaration	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 87	Java Problem
* Javadoc: Missing comment for public declaration	Java9Support.java	/org.jacoco.core/src/org/jacoco/core/internal	line 94	Java Problem